recipes-tag/mayavi/yum_requirements.txt

```diff
@@ -0,0 +1,3 @@
+libXt-devel
+mesa-libGLU-devel
+xorg-x11-server-Xvfb
```

Comment:
Not sure if this file will be taken into account. Is it something conda can install? Or is it handled by some extra scripts?

We are on the `apt`-based system, probably can have something like this.

Reply:
I don't know if we need this. At CF we add it to our CI scripts explicitly, but you might have these on your build system which match the systems on the floor.

Reply:
I see. Let's keep it for now, it has no effect.